src/activities/jumbled_words/ActivityInfo.qml

```diff
@@ -0,0 +1,37 @@
+/* GCompris - ActivityInfo.qml
+ *
+ * Copyright (C) 2016 Komal Parmaar <parmaark@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import GCompris 1.0
+
+ActivityInfo {
+  name: "jumbled_words/Jumbled_words.qml"
+  difficulty: 1
+  icon: "jumbled_words/jumbled_words.svg"
+  author: "Komal Parmaar &lt;parmaark@gmail.com&gt;"
+  demo: false
```

Comment:
This should be set to true. This property tells if the activity is accessible by default on the gratis version or not.